packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardBuilder.ts

```diff
@@ -104,7 +106,13 @@ export default class AdaptiveCardBuilder {
   }
 
   addCommonHeaders(content: ICommonContent) {
-    this.addTextBlock(content.title, { color: TextColor.Dark, size: TextSize.Medium, weight: TextWeight.Bolder });
+    const { richCardsWrapTitle } = this.styleOptions;
+    this.addTextBlock(content.title, {
+      color: TextColor.Dark,
+      size: TextSize.Medium,
+      weight: TextWeight.Bolder,
+      wrap: richCardsWrapTitle
+    });
     this.addTextBlock(content.subtitle, { color: TextColor.Dark, isSubtle: true, wrap: true });
```

Comment:
I think we should also use the same `richCardsWrapTitle` for `subtitle`, because subtitle is a type of title too.

Reply:
Would it make more sense to add `richCardWrapSubtitle` to the default style options? I think right now subtitles are designed to wrap and titles don't. Using `richCardWrapTitle` for both might change Web Chat's card behavior too much.